DNN Platform/Library/Entities/Modules/ModuleController.cs

```diff
@@ -1382,9 +1382,34 @@ public ArrayList GetAllTabsModulesByModuleID(int moduleID)
         /// get a Module object
         /// </summary>
         /// <param name="moduleID">ID of the module</param>
+        /// <returns>ModuleInfo object</returns>
+        /// <remarks>This overload ignores any cached values and always retrieves the latest data
+        /// from the database.</remarks>
+        public ModuleInfo GetModule(int moduleID)
+        {
+            return GetModule(moduleID, Null.NullInteger, true);
```

Comment:
I understand that this is just reinstating the obsolete method, but is there a good reason to have this API ignore the cache by default?  Shouldn't you have to make an explicit request to go around the cache?